src/libraries/Native/AnyOS/entrypoints.h

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include <stdint.h>
+#include <string.h>
+
+#ifndef lengthof
+#define lengthof(rg) (sizeof(rg)/sizeof(rg[0]))
+#endif
+
+typedef struct
+{
+    const char* name;
+    const void* method;
+} Entry;
+
+// expands to:      {"impl", (void*)impl},
+#define OverrideEntry(impl) \
```

Comment:
```suggestion
#define DllImportEntry(impl) \
```
There is no other mention of "override" in this file, so override looks out of place.

Reply:
Changed to DllImportEntry.